components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/service_constants.h

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_VPN_BROWSER_CONNECTION_WIREGUARD_WIN_BRAVE_VPN_WIREGUARD_SERVICE_SERVICE_CONSTANTS_H_
+#define BRAVE_COMPONENTS_BRAVE_VPN_BROWSER_CONNECTION_WIREGUARD_WIN_BRAVE_VPN_WIREGUARD_SERVICE_SERVICE_CONSTANTS_H_
+
+#include <guiddef.h>
+#include <string>
+
+namespace brave_vpn {
+
+constexpr wchar_t kBraveWireguardServiceExecutable[] =
```

Comment:
I assume we want our executable to have `brave_vpn` in the name? If that's the case, then let's have all related variables also include "Vpn" in the name:
`kBraveWireguardServiceExecutable` -> `kBraveVpnWireguardServiceExecutable`

It that's not the case, then we can name the executable `brave_wireguard_service`, and use `brave_wireguard` prefix everywhere. Right now we have `LIBID_BraveVpnWireguardServiceLib`, but `IID_IBraveWireguardManager` which is not very consistent.